docs/Api Specifications.yaml

```diff
@@ -451,8 +451,10 @@ definitions:
           - 4k
           - laser
           - 4dx
+          - screenx
           - dolbycinema
           - dolbyatmos
+          - regularshowing
```

Comment:
Why name this variable regular*showing* instead of just _regular_?

Reply:
Because filtering for `regular` sounded too ambiguous to me?

Reply:
ah yea, that makes sense!